Module not found: Default condition should be last one

The latest version 3.1.1 is throwing the below error. No issues with 3.1.0 in NextJS 13.5. 

Module not found: Default condition should be last one
> 2 | import { AssemblyAI } from 'assemblyai';
  3 | import { NextRequest, NextResponse } from "next/server";

--- a/package.json
+++ b/package.json
@@ -1,16 +1,16 @@
 {
   "name": "assemblyai",
-  "version": "3.1.1",
+  "version": "3.1.2",
   "description": "The AssemblyAI Node.js SDK provides an easy-to-use interface for interacting with the AssemblyAI API, which supports async and real-time transcription, as well as the latest LeMUR models.",
   "main": "dist/index.js",
   "module": "dist/index.esm.js",
   "types": "dist/index.d.ts",
   "typings": "dist/index.d.ts",
   "exports": {
-    "default": "./dist/index.js",
     "require": "./dist/index.js",
     "import": "./dist/index.esm.js",
-    "node": "./dist/index.js"
+    "node": "./dist/index.js",
+    "default": "./dist/index.js"
   },
   "repository": {
     "type": "git",

